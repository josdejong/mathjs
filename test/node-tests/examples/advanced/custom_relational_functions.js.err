"a" == "b" Error: Cannot convert "a" to a number
"a" == "a" Error: Cannot convert "a" to a number
