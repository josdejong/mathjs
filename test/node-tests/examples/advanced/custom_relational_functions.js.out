default (compare string by their numerical value)
2 < 10 true
"2" < "10" true

custom (compare strings lexically)
2 < 10 true
"2" < "10" false
"a" == "b" false
"a" == "a" true
